### ISSUE TITLE
Hard coding

### ISSUE THREAD

**Issue by starbuck5250** [starbuck5250/SEUEXIT#5]
Currently, the CLP that creates the message file is hard coded for library BUCK.  That's not necessary; we can make it a parameter.

**Comment by worksofliam** [comment]
Yes, please make it a parameter. Once this is changed, the build file can change to call `SEUEXITCL` with the library parameter. 

[build.txt at line 13](https://github.com/starbuck5250/SEUEXIT/blob/master/build.txt#L13)

Currently
`CALL &INTOLIB/SEUEXITCL`

After your CLP change
`CALL &INTOLIB/SEUEXITCL PARM('&INTOLIB')`

**Comment by starbuck5250** [comment]
I uploaded a branch called SEUEXIT-rdi_egit which fits the way I work with RDi + iProjects.  Relic package manager is flexible enough to allow for this; I just changed build.txt to handle the folder hierarchy.  The CLP and build script now use a parameter for the destination library.

RDi has this kind of folder hierarchy:

```
SEUEXIT
  QCLSRC
    SEUEXITCL.CLP
  QRPGLESRC
    SEUEXIT.RPGLE
```

This has one distinct advantage to a flatter style: One can name a *CMD object the same as a *PGM.  So I could (for example) have a RELICGET command whose command processing program is also called RELICGET.  This is particularly important to me because I create a /copy member, service program, binder source, and source member all with the same name, and a flat folder hierarchy wouldn't easily support that.
